mailpoet/RoboFile.php

```diff
@@ -111,6 +111,36 @@ public function translationsBuild() {
       )->run();
   }
 
+  public function translationsGetPotFileFromCircleCI() {
```

Comment:
It is just a small detail but I would suggest using a different name here e.g. `translationsGetPotFileFromReleaseZip`.
Currently, we download the release zip from CircleCi, but we may change the service or we decide to download the .zip from the release PR on Github so the name and also the error message may become obsolete/confusing.

Reply:
Fair point, @costasovo. What do you think of `translationsGetPotFileFromBuild()`?